descriptive-link-5effbb.md

```diff
@@ -0,0 +1,179 @@
+---
+id: 5effbb
+name: Link text is descriptive
+rule_type: atomic
+
+description: | 
+ This rule checks that all links describe their purpose
+
+accessibility_requirements: 
+  wcag20:2.4.4: # Link Purpose (In Context) 
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+
+input_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Carlos Duarte
+- Marie Trudelle 
+- Ramit Garg
+---
+
+## Applicability
+
+The rule applies to any HTML element with the [semantic role](#semantic-role) of `link` that has an [accessible-name](#accessible-name) that is not the empty string ("").
```

Comment:
Comparing with https://act-rules.github.io/rules/b20e66, should it be?

The rule applies to any HTML **_or SVG_** element with the [semantic role](#semantic-role) of `link`**_, or a role that inherits from the link role, are  [included in the accessibility tree](#included-in-the-accessibility-tree)_**, and that has an [accessible-name](#accessible-name) that **_do not only consist of [whitespace](#whitespace)_**.